CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.7.1
+
+### 🐞 Bug fixes
+
+* Disable antialias for 15.4/15.5 iOS devices and M1 Macbooks using 15.4/15.5 Safari to resolve a [WebKit rendering bug](https://bugs.webkit.org/show_bug.cgi?id=237918#c3) ([#11615](https://github.com/mapbox/mapbox-gl-js/pull/11615))
```

Comment:
Since we can't detect M1 we disable it for Safari 15.4 and 15.5 everywhere, not just on M1. While this also affects Firefox and Chrome on iOS, I think we can just refer to "Safari" here since they are essentially the same as Safari.

I'd also expand "antialias" to "WebGL context antialiasing" here

Reply:
Just saw your suggestion for this one and committe dit